package.json

```diff
@@ -1,39 +1,45 @@
 {
   "name": "aurelia-open-id-connect",
-  "description": "An Aurelia adapter for the IdentityModel/oidc-client-js.",
   "version": "0.22.0",
+  "private": true,
+  "bugs": {
+    "url": "https://github.com/shaunluttin/aurelia-open-id-connect/issues"
+  },
+  "license": "MIT",
   "author": "Shaun Luttin <shaun@bigfont.ca>",
-  "main": "dist/amd/index.js",
-  "types": "dist/amd/index.d.ts",
+  "main": "dist/commonjs/index.js",
+  "typings": "dist/commonjs/index.d.ts",
```

Comment:
Why the change from `types` to `typings`?

As far as I can tell from the documentation, `types` and `typings` are synonymous. https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

> Note that the "typings" field is synonymous with "types", and could be used as well.

Reply:
You are right, what i did is take the files from https://github.com/aurelia/ux as the package base.
Thats the only reason.